src/portable_python/cpython.py

```diff
@@ -104,6 +104,9 @@ def url(self):
         if PPG.config.get_value("cpython-use-github"):
             return f"https://github.com/python/cpython/archive/refs/tags/v{self.version}.tar.gz"
 
+        if cfg_url := self.cfg_url(self.version):
```

Comment:
We need py3.8+ for `:=`, we need to adapt `python_requires` to reflect this in `setup.py`, and remove the legacy tests, ie the `test-eol:` section from `.github/workflows/tests.yml`